### PR TITLE
Editorial c

### PR DESCRIPTION
**Pull request by paul-grassi**
This should resolve all but a few editorial edits in -C.

### CODE REVIEW

File: sp800-63-3/definitions.md [hunk 1]
```diff
@@ -43,6 +43,9 @@ A party who acts with malicious intent to compromise a system.
 #### Attribute
 A quality or characteristic ascribed to someone or something.
 
+#### Attribute Bundle
+A packaged set of attributes, usually contained within an assertion.  Attribute bundles offer relying parties a simple way to retrieve the most relevant attributes they need from Identity Providers.  Attribute bundles are synonymous with OpenID Connect scopes. See [OpenID Connect Core 1.0](http://openid.net/specs/openid-connect-core-1_0.html#ScopeClaims).
```
Comment (jimfenton) [inline sp800-63-3/definitions.md]: "identity providers" should be lower case for consistency.

File: sp800-63c/sec4_federation.md [hunk 2]
```diff
@@ -110,8 +110,8 @@ The following requirements apply specifically to agencies:
...
 
 Due to the distributed nature of a federated system, the subscriber is capable of terminating sessions with the IdP and RP independently of one another. The RP SHALL NOT assume that the subscriber has an active session at the IdP past the establishment of the federated log in. The IdP SHALL NOT assume that termination of the subscriber's session at the IdP will propagate to any sessions that subscriber would have at downstream RPs.
 
-See [Section 7 of 800-63B](sp800-63b.html#sec7) for more information about session management requirements. 
+See [Section 7 of 800-63B](sp800-63b.html#sec7) for more information about session management requirements.
```
Comment (jimfenton) [inline sp800-63c/sec4_federation.md]: Should be "See SP 800-63B Section 7"

File: sp800-63c/sec5_assertions.md [hunk 3]
```diff
@@ -5,40 +5,43 @@
...
 
 Although details vary based on the exact federation protocol in use, an assertion SHOULD be used only to represent a single login event at the RP. After the RP consumes the assertion, [session management](sp800-63b.html#sec7) by the RP comes into play; the assertion SHALL NOT be used past the expiration time contained therein. However, the expiration of the session at the RP MAY occur prior to the expiration of the assertion. See [Section 4.3](#federation-session) for more information.
 
 ### 5.1. Assertion Binding
 
-An assertion can be classified based on whether presentation by a claimant of an assertion reference or the assertion itself is sufficient for establishing the binding between the subscriber and the assertion, or if a stronger binding is required.
+ Assertion binding can be classified based on whether:
+ - presentation by a claimant of an assertion reference; or
```
Comment (jimfenton) [inline sp800-63c/sec5_assertions.md]: Breaking this up into bullets makes this read unevenly. The first bullet doesn't stand alone.

File: sp800-63c/sec7_fal.md [hunk 4]
```diff
@@ -5,11 +5,11 @@
...
+This section defines allowable Federation Assurance Levels, or FAL. The FAL describes requirements for how assertions and constructed and secured for a given transaction. These levels can be requested by an RP or required by configuration of both RP and IdP for a given transaction. 
 
-The FAL combines aspects of [assertion protection](#sec5) and [assertion presentation](#sec6) into an ordinal measurement scale applicable across different [federation models](#sec4). All assertions SHALL comply with the requirements in [Section 5](#sec5). While many other combinations of factors are possible, this list is intended to provide clear implementation recommendations representing increasingly secure deployment choices. Combinations of aspects not found in the FAL table are possible but outside the scope of this document.
+All assertions SHALL comply with the detailed requirements in [Section 5](#sec5). While many other combinations of factors are possible, this list is intended to provide clear implementation recommendations representing increasingly secure deployment choices. Combinations of aspects not found in the FAL table are possible but outside the scope of this document.
```
Comment (jimfenton) [inline sp800-63c/sec7_fal.md]: What does "factors" refer to?

**Review by jimfenton — CHANGES_REQUESTED** [review]
Mostly minor issues, but one significant readability problem.

**Comment by paul-grassi** [comment]
@jimfenton ready for you again.  Thanks!

**Review by jimfenton — APPROVED** [review]

**Pull request merged**